scripts/pip-compile.sh

```diff
@@ -8,8 +8,22 @@ if ! python -c "import sys; assert sys.version_info.major == $major and sys.vers
   exit 1
 fi
 
+# NOTE(alan): Order matters in compiling. Start with base and test. Then do the rest in any order.
```

Comment:
this PR (https://github.com/Unstructured-IO/unstructured/pull/1295) might help with this situation